src/domUpdates.js

```diff
@@ -146,6 +176,10 @@ dashboardMenuButton.addEventListener('click', () => {
 
 filterButton.addEventListener('click', domUpdates.populateAvailableRooms);
 
-
+availableRoomsSection.addEventListener('click', (e) => {
```

Comment:
Can the logic of this eventListener go just inside the function of BookRoom(e) instead of in the anonymous function??? 

I remember end of Mod 1 some of the feed back was not to do this with the exception of testing for 
**if (event.key === 'Enter')**

Reply:
Great point, I was wondering where it made the most sense to have that logic. It should be pretty simple to move it into the bookRoom function.